### ISSUE TITLE
Minimum version of Moodle

### ISSUE THREAD

**Issue by hgeorgsch** [KQMATH/moodle-mod_jazzquiz#39]
This is half of #31 which was closed when the bugs with Moodle 3.1 were resolved.
We need to make a final release supporting Moodle 3.1 ASAP.
When that's done the minimum version should be changed to 3.5, to avoid having to continue support on old versions of Moodle.

**Comment by sebastsg** [comment]
This will be version 0.9.1, released later today. Closing this after that.

**Comment by hgeorgsch** [comment]
No. Before you change it, the version requirement must be changed on the future development branch, so that 0.9.1 be the last release supporting 3.1-3.4. I am worried we forget.

**Comment by hgeorgsch** [comment]
Changed to require moodle 3.5 on master.
